src/sass/extends/_charts-and-graphs.scss

```diff
@@ -3,6 +3,7 @@
   display: flex;
   justify-content: center;
   align-items: center;
+  max-width: 350px;
```

Comment:
- Small CSS fix, looks good.
- We don't have this implemented on our product as of now. OK to merge.